docs/topics/native/native-c-interop.md

```diff
@@ -646,4 +646,55 @@ fun readData(fd: Int) {
 ```
 
 Here we use service function `usePinned`, which pins an object, executes block and unpins it on normal and
-exception paths.
\ No newline at end of file
+exception paths.
+
+### Forward declarations
+
+To import forward declarations, use the `cnames` package. For example, to import a `cstructName` forward declaration
+declared in a C library with a `library.package`, use a special forward declaration package:
+`import cnames.structs.cstructName`.
+
+To transfer objects between libraries, you need to make an explicit `as` cast to and from the corresponding C
```

Comment:
> To transfer objects between libraries, you need to make an explicit `as` cast 

This is a bit imprecise and might be misleading. For example, if both libraries have only forward declarations for `cstructName`, then you don't need any cast.

I propose rephrasing this part the following way:
1. First, introduce the second library as well, and mention that it has a real declaration that goes into another package.
2. And after that, you can say that transferring objects between these two libraries requires a cast.

Reply:
The same applies to the Objective-C page.

Reply:
Thank you! Moved this sentence after the second lib